pkg/iptables/rules.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright © 2022 SUSE LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package iptables
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/Masterminds/log-go"
+)
+
+const (
+	AppendToChain   = "append"
```

Comment:
Exported things should have comments (even though it feels silly in this case).

Should we declare a type (alias of `string`) for `chainOP` to make it more obvious that these are the values we expect?